### ISSUE TITLE
Usage with Next.js

### ISSUE THREAD

**Issue by janhesters** [dequelabs/axe-core-npm#103]
Next.js handles React and React DOM for you. How would you add this to a Next.js project?

**Comment by jeeyyy** [comment]
@janhesters which package are you looking to use with `next.js`. Based on that we may be able to help.

**Comment by stephenmathieson** [comment]
I assume you're asking about the `@axe-core/react` package. If so, I'd add the hook (described in the package's README.md) in a `pages/_app.js` file. It's been awhile since I've used Next.js, but I believe that's as close to the `ReactDOM.render()` call we can get.

I'm now generally curious about this, so may put an example together within the next few days. Feel free to ping me here if you don't get an update soon.

**Comment by juwanpetty** [comment]
Hey, just used this implementation in my `create-next-app` and am getting the error: `Error was not caught TypeError: conf is undefined` in my console when I run the development server.

**Comment by stephenmathieson** [comment]
Hey @juwanpetty! I believe that may have been fixed by the work @Alex-Joseph did in https://github.com/dequelabs/axe-core-npm/pull/188. I don't think the patch has been officially released, but to try it out, you could install the `@next` version of `react-axe`:

```sh
cd yourapp
npm install --save @axe-core/react@next
```

If that doesn't solve the problem, then we should open a new ticket for it, as I don't _think_ it's specific to Next.js.

Let me know!

**Comment by stephenmathieson** [comment]
From looking at [this CI build](https://app.circleci.com/pipelines/github/dequelabs/axe-core-npm/449/workflows/2793b59c-8b20-4cee-9d83-75200eaf6151/jobs/3244), it looks like you'll want to try out `@axe-core/react@4.1.1-alpha.89+708c463` (which is available with the `@next` dist tag).

**Comment by juwanpetty** [comment]
Thank you @stephenmathieson that fixed my problem 😊
